ios/brave-ios/App/Client.xcodeproj/Brave.xctestplan

```diff
@@ -40,6 +40,9 @@
       }
     },
     {
+      "skippedTests" : [
```

Comment:
Ah why are we skipping them? Wallet doesnt need them anymore? Can we delete the file ?

Reply:
We can't use this test due to Brave-Core. The test requires using the URIFixup class which now uses the Brave-Core AutoComplete classifier. However, Brave-Core can't be used in Unit Tests atm. So the test is disabled.